devkit/loadtesting-k6/package.json

```diff
@@ -12,9 +12,13 @@
         "lint:ci": "eslint --max-warnings 0 src/",
         "test": "exit 0",
         "help": "node build/rpc-server/help.js",
-        "start:discovery-platform": "k6 run src/discovery-platform/artificial.js --out json=results/results-discovery-platform.json"
+        "start:discovery-platform": "k6 run src/discovery-platform/artificial.js --out json=results/results-discovery-platform.json",
+        "docker:build": "cd ../../ ; docker build --platform linux/amd64 --tag europe-west6-docker.pkg.dev/rpch-375921/rpch/rpch-k6:latest -f devkit/loadtesting-k6/Dockerfile .",
+        "docker:push": "docker push europe-west6-docker.pkg.dev/rpch-375921/rpch/rpch-k6:latest"
     },
     "devDependencies": {
-        "@types/k6": "^0.47.1"
+        "@types/k6": "^0.47.1",
+        "@types/node": "^20.8.5",
+        "typescript": "^5.2.2"
```

Comment:
Do you actually need this?
The typescript version comes from the root package.json as part of the monorepo dependencies.

Reply:
I couldn't make it compile de Dockerfile

Reply:
Your approach tries to ignore the mono repo structure.

If you look into the other apps and packages, they all have Dockerfiles that you could use as a baseline.